src/site/content/en/blog/es-modules-in-sw/index.md

```diff
@@ -0,0 +1,179 @@
+
+## Current limitations
+
+### Static imports only
+
+ES modules can be imported in one of two ways: either
+[statically](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import),
+using the `import ... from '...'` syntax, or
+[dynamically](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports),
+using an `import('...')` statement. Inside of a service worker, only the static
+syntax is currently supported.
+
+This limitation is analogous to a
+[similar restriction](https://developers.google.com/web/updates/2018/10/tweaks-to-addAll-importScripts)
+placed on `importScripts()` usage. Dynamic calls to `importScripts()` do not
+work inside of a service worker, and all `importScripts()` calls, which are
+inherently synchronous, must complete before the service worker completes its
+`install` phase. This restriction ensures that the browser knows about, and is
+able to implicitly cache, all JavaScript code needed for a service worker's
+implementation during installation.
+
+At some point in the future, this restriction might be lifted, and dynamic ES
+module imports
+[may be allowed](https://github.com/w3c/ServiceWorker/issues/1356#issuecomment-783220858).
+For the time being, though, ensure that you only use the static syntax inside of
+a service worker.
+
+#### What about other workers?
+
+Support for
+[ES modules in "dedicated" workers](https://web.dev/module-workers/)—those
+constructed with `new Worker('...', {type: 'module'})`—is more widespread, and
+has been supported in Chrome and Edge since
+[version 80](https://chromestatus.com/feature/5761300827209728), as well as
+[recent versions](https://bugs.webkit.org/show_bug.cgi?id=164860) of Safari.
+Both static and dynamic ES module imports are supported in dedicated workers.
+
+Chrome and Edge have supported ES modules in
+[shared workers](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker)
+since [version 83](https://chromestatus.com/feature/5169440012369920), but no
+other browser offers support at this time.
+
+### No support for import maps
+
+[Import maps](https://github.com/WICG/import-maps/blob/main/README.md) allow
+runtime environments to rewrite module specifiers, to, for example, prepend the
+URL of a preferred CDN from which the ES modules can be loaded.
+
+While Chrome and Edge
+[version 89](https://www.chromestatus.com/feature/5315286962012160) and above
+support import maps, they currently
+[cannot be used](https://github.com/WICG/import-maps/issues/2) with service
+workers.
+
+## Browser support
+
+ES modules in service workers are supported in Chrome and Edge starting with
+[version 91](https://chromestatus.com/feature/4609574738853888).
+
+Safari added support in the
+[Technology Preview 122 release](https://webkit.org/blog/11577/release-notes-for-safari-technology-preview-122/#:~:text=Added%20support%20for%20modules%20in%20Service%20Workers),
```

Comment:
```suggestion
[Technology Preview 122 Release](https://webkit.org/blog/11577/release-notes-for-safari-technology-preview-122/#:~:text=Added%20support%20for%20modules%20in%20Service%20Workers),
```

Reply:
As a writer I think Apple is wrong in capitalizing 'release'. But they did and the rule is to follow their practice.